tests/spec/core/inlines-spec.js

```diff
@@ -224,6 +224,36 @@ describe("Core - Inlines", () => {
     expect(badOne.textContent).toBe("[[[#does-not-exist]]]");
   });
 
+  it("processes normative and informative `[[[spec/path#fragment]]]` into a link with the correct data-cite and text content", async () => {
+    const body = `
+      <p id="output">
+        [[[!HTML/path.html#some-normative-fragment]]]
+        [[[?HTML/path.html#some-informative-fragment]]]
+      </p>
+    `;
+
+    const doc = await makeRSDoc(makeStandardOps(null, body));
+    const anchors = doc.querySelectorAll("#output a");
+
+    expect(anchors.length).toBe(2);
+
+    const [normativeAnchor, informativeAnchor] = anchors;
+
+    expect(normativeAnchor.textContent).toBe(
```

Comment:
Maybe skip the normative/informative part in tests. We're testing the inline section like syntax, not whether normative/informative works here.
```suggestion
    expect(anchors[0].textContent).toBe(
```

Reply:
Hm. I like naming it because it explains the different cases we're catering for and why we support them. In this case, the cases are `[[[spec/path#frag]]]`, `[[[!spec/path#frag]]]` and `[[[?spec/path#frag]]]` without us having to explicitly say so.

^ ...and that third case with no `?` or `!` doesn't have a test. Time to add it.

Reply:
@sidvishnoi  right, we only care about output in this part, not how the reference is classified. We test ? ! classifications in other tests so testing for those is redundant here.

Reply:
> We test ? ! classifications in other tests
ah okay. wasn't aware about that. i've renamed the variables accordingly.